pages/users/settings.tsx

```diff
@@ -0,0 +1,104 @@
+import { useState } from 'react';
+import Layout from 'components/Layout';
+import { Button, Link, TextField } from '@material-ui/core';
+import ProgressStepper from 'components/user/ProgressStepper/index';
+import EmailVerify from 'components/user/EmailVerify/index';
+import { sampleUserData } from 'utils/sample-data';
+import styles from '../../styles/users/Settings.module.css';
+
+function capitalizeFirstLetter(string: string): string {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+const UserProfSettings: React.FC = () => {
+  const [setting, setSetting] = useState(0);
+  const hiddenPassword = '******';
+  const emailButton =
+    setting === 0 ? (
+      <div className={styles.field}>
+        <div>Email</div>
+        <div className={styles.fieldRight}>
+          {sampleUserData[0].email}
+          <Button
+            variant="outlined"
+            color="primary"
+            disableElevation
+            onClick={() => setSetting(1)}
+            className={styles.fieldButton}
+          >
+            Edit
+          </Button>
+        </div>
+      </div>
+    ) : (
+      <div className={styles.field}>
+        <div>Email</div>
+        <div className={styles.fieldRight}>
+          <TextField
+            id="email"
+            defaultValue={sampleUserData[0].email}
+            variant="outlined"
+            size="small"
+          />
+          <Button
+            className={styles.fieldButton}
+            variant="contained"
+            color="primary"
+            disableElevation
+            onClick={() => setSetting(0)}
+          >
+            Save
+          </Button>
+        </div>
+      </div>
+    );
+
+  const passwordButton = (
+    <div className={styles.passwordButton}>
+      <div className={styles.field}>
+        <div>Password</div>
+        <div className={styles.fieldRight}>
+          {hiddenPassword}
+          <Link href="../users/password_change">
+            <Button
+              variant="outlined"
+              color="primary"
+              disableElevation
+              className={styles.fieldButton}
+            >
+              Edit
+            </Button>
+          </Link>
```

Comment:
use router to link
https://github.com/calblueprint/nbjc/blob/a51a7af389736ee893665f6727ace20bf1a205b3/components/Layout.tsx#L46